api/node/rust/interpreter/component_instance.rs

```diff
@@ -137,9 +137,14 @@ impl JsComponentInstance {
             self.inner
                 .set_callback(callback_name.as_str(), {
                     let return_type = return_type.clone();
+                    let callback_name = callback_name.clone();
 
                     move |args| {
-                        let callback: JsFunction = function_ref.get().unwrap();
+                        let Ok(callback) = function_ref.get::<JsFunction>() else {
+                            eprintln!("Ndoe.js: callback {} throws an exception", callback_name);
+                            return Value::Void;
```

Comment:
I think that this might cause panics in the interpreter if the return value of the Slint callback isn't void. I think void should only be returned if there is no return_type or if the return_type is void, otherwise we should return value for the return type. Fortunately default_value_for_type handles that.

Please also add a test that covers this scenario.